COMMUNICATIONS.md

```diff
@@ -0,0 +1,94 @@
+# OpenSearch Project Communication 
+
+- [Overview](#overview)
+- [Slack](#public-slack)
+  - [Getting Started](#getting-started)
+  - [Workspace Channels](#workspace-channels)
+  - [Tips](#tips)
+
+## Overview
+
+The purpose of this document is to provide references and information regarding
+the communication channels for the OpenSearch Project. All communication is
+subject to the [OpenSearch Code of Conduct](CODE_OF_CONDUCT.md). Please see
+[CONTRIBUTING](CONTRIBUTING.md) if you're interested in contributing to the
+project.
+
+## Slack
+
+OpenSearch has a workspace on [Slack](https://opensearch.slack.com) to provide
+open communication channels for all people interested in OpenSearch.
+
+The following guidelines include a Getting Started for steps to register and
+setup the workspace along with tips for using the workspace. Please read through
+these guidelines carefully and thoroughly. All communications and behavior on
+the public slack workspace is subject to the 
+[OpenSearch Code of Conduct](CODE_OF_CONDUCT.md).
+
+### Getting Started
+
+_Join Us_
+
+  * Register your account at https://opensearch.slack.com. You will be required
+    to enroll in two-factor authentication to ensure the best possible security
+    of your account.
+
+_Update Profile_
+
+  * You may add your interests or keywords in the “What I do” section of your 
+    profile along with your Title and Company/Organization. This helps 
+    participants network and connect with each other by common interests. No 
+    anonymous display names are allowed.
+  * Feel free to update your profile photo to something unique to you. All 
+    profile photos are subject to the 
+    [OpenSearch Code of Conduct](CODE_OF_CONDUCT.md) policies.
```

Comment:
Again COC, just remove this and call it once above.

Reply:
My worry is that it would imply that other parts aren't subject to CoC.

Reply:
I would agree with @nknize here - best to err on the side of caution. We've had issues with this in Meetup to the point of completely disabling profile pictures.